analysis/report_templates/default.html

```diff
@@ -16,7 +16,12 @@
 <!DOCTYPE html>
 <html>
     <head>
-        <title>FuzzBench: {{ experiment.name }} report</title>
+        <title>
```

Comment:
I think the warning needs to be much more prominent than "(wip)" or else no one will notice.

Reply:
That is in title of tab. More prominent one is in big title on page, see change below.

Reply:
![Screenshot from 2020-03-23 15-04-04](https://user-images.githubusercontent.com/8731250/77367609-c4b39400-6d17-11ea-9002-1b350530f930.png)

Reply:
1. I wouldn't use the abbreviation, write out "work in progress"
2. I don't think it's obvious to people what it means for a report to be in progress so explaining this somewhere would help.